src/nvim/version.c

```diff
@@ -80,8 +80,8 @@ static int included_patches[] = {
   784,
   //783,
   //782,
-  //781,
-  //780,
+  781,
+  //780 NA
```

Comment:
Because it doesn't apply to our codebase anymore. The variables moved and are initialized already. Most of these "fix compiler warning" patches will be marked as `NA`, since our CI catches them.

https://code.google.com/p/vim/source/detail?r=9a13e3910e3d45c768a1e3d87a15a62b4cee439c